src/hook.ts

```diff
@@ -92,6 +92,7 @@ export const useStorage = <T = any>(rawKey: RawKey, onInit?: Setter<T>) => {
     return () => {
       isMounted.current = false
       storageRef.current.unwatch(watchConfig)
+      if (onInit instanceof Function) setRenderValue(onInit)
```

Comment:
nit: let's put this under block inside curly brackets to keep it consistent with the repo